addons/MetroidvaniaSystem/Database/MapEditor.tscn

```diff
@@ -627,6 +627,15 @@ func create_scene_confirm() -> void:
 	EditorInterface.get_resource_filesystem().scan()
 	EditorInterface.open_scene_from_path(scene_path)
 	on_map_selected(scene_path)
+
+
+func can_drop_data(at_position: Vector2, data) -> bool:
+	return data.type == \"files\" and len(data.files) == 1 and not highlighted_room.is_empty()
```

Comment:
It's safer to assume that `"type"` will now always exist and use `data.get("type", "") == "files"`.

Reply:
Fixed, and added root folder check. Condition was getting a bit complicated so I refactored to guard clauses to make it more readable.